.npmignore

```diff
@@ -8,4 +8,3 @@ build.js
 .jshintrc
 .editorconfig
 /.idea/
-/test/
```

Comment:
we should exclude test folders from beeing pushed to npm. we should do that for all canjs projects.

Reply:
We can't do that right now because canjs/canjs loads the ajax tests. I agree with you though, we just need a new way to do the canjs/canjs stuff in that case.